### ISSUE TITLE
LC0005 "There was an Error in Rule "LC0005" of type "missing 'XmlVersionNo' ordinals.""

### ISSUE THREAD

**Issue by rvanbekkum** [StefanMaron/BusinessCentral.LinterCop#1000]
Hi, I would like to report an exception reported by LC0000.

If you have an `xmlport` that uses the `XmlVersionNo = V10;` property setting, then the following diagnostic is raised by LC0000:

```
There was an Error in Rule "LC0005" of type "missing 'XmlVersionNo' ordinals." Help improving this rule and open a GitHub issue, pretty please? :-)
```

**Comment by Arthurvdv** [comment]
Thank you for reporting this 🙋‍♂️

Fixed this type with https://github.com/StefanMaron/BusinessCentral.LinterCop/commit/225ae541eb7a4faac41b97cfa7cbde12b1b2fc93, which is now available in the pre-release version of the LinterCop.

**Comment by Arthurvdv** [comment]
The version [v0.36.0](https://github.com/StefanMaron/BusinessCentral.LinterCop/releases/tag/v0.36.0) of LinterCop has been released as the latest, where this issue is now resolved.
